src/pages/PageSearchResult.vue

```diff
@@ -87,37 +96,54 @@ export default {
       loading: false,
       post: null,
       error: null,
+      more: false,
+      results: { dinos: [], total: 0, next: null },
     };
   },
   created() {
-    this.fetchData();
+    this.fetchNew();
   },
   watch: {
-    $route: 'fetchData',
+    $route: 'fetchNew',
   },
   methods: {
-    async fetchData() {
+    async fetchNew() {
+      this.loading = true;
+      const results = await this.fetchData();
+      this.results = results;
+      this.loading = false;
+    },
+    async fetchMore() {
+      this.loading = true;
+      const results = await this.fetchData(true);
+      this.results.dinos.push(...results.dinos);
+      this.results.next = results.next;
+      this.loading = false;
```

Comment:
Quick idea: maybe at this point we could convey that an update happened through a `Toast`?

Reply:
There's a loading spinner when loading. I think an additional toast would be overkill here.